mmv1/templates/terraform/examples/vertex_ai_featureonlinestore_with_bigtable.tf.erb

```diff
@@ -0,0 +1,16 @@
+##FeatureOnlineStore With BigTable
+resource "google_vertex_ai_feature_online_store" "<%= ctx[:primary_resource_id] %>" {
+  name     = "<%= ctx[:vars]['name'] %>"
+  labels = {
+    foo = "bar"
+  }
+  region   = "us-central1"
+  bigtable {
+    auto_scaling {
+        min_node_count = 1
+        max_node_count = 3
+        cpu_utilization_target = 50
+    }
+  }
+  force_destroy = true
```

Comment:
This will show up on our website as an example, so I just wanted to make sure, do we want to instruct users to set `force_destroy = true` by default? Since it is false by default, my assumption for a field like this would be for users to use caution when setting it true.

Reply:
Yeah, You're right. The intention was to show that this is a field that could be set in one of the examples, but since it would be in the documentation anyway, I've removed it now.